src/wooting-usb.h

```diff
@@ -55,16 +71,27 @@ typedef struct _KeyboardMatrixID {
 #define WOOTING_RESET_ALL_COMMAND 32
 #define WOOTING_COLOR_INIT_COMMAND 33
 
-void wooting_usb_set_disconnected_cb(void_cb cb);
-void wooting_usb_disconnect(bool trigger_cb);
+WOOTINGRGBSDK_API void wooting_usb_set_disconnected_cb(void_cb cb);
+WOOTINGRGBSDK_API void wooting_usb_disconnect(bool trigger_cb);
 
-bool wooting_usb_find_keyboard(void);
+WOOTINGRGBSDK_API bool wooting_usb_find_keyboard(void);
```

Comment:
I think we want to avoid intentionally exposing the `wooting_usb_set_disconnected_cb`, `wooting_usb_find_keyboard`, `wooting_usb_get_meta` functions as they have corresponding public facing functions in the `wooting-rgb-sdk.h`